packages/cozy-harvest-lib/src/components/AccountLogoutButton.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import omit from 'lodash/omit'
+
+import { Button } from 'cozy-ui/react/Button'
+import { translate } from 'cozy-ui/react/I18n'
+import { withMutations } from 'cozy-client'
+
+import accountsMutations from '../connections/accounts'
+
+export class AccountLogoutButton extends Component {
```

Comment:
As we talk before: I realy don't like the component name here.

I suggest `DeleteAccountButton` instead.